### ISSUE TITLE
L1 warnings in premix stage 1

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cmssw#17923]
When running a command like this in 9_0_0_pre5 or pre6, for 2017 or 2018, using pre5 minbias relval\* as input:
```
cmsDriver.py stage1_premix --conditions auto:phase1_2017_realistic --pileup_input file:a.root -n 100 --era Run2_2017 --eventcontent PREMIX -s GEN,SIM,DIGIPREMIX,L1,DIGI2RAW --datatier GEN-SIM-DIGI-RAW --pileup AVE_35_BX_25ns --evt_type SingleNuE10_cfi --fileout file:premix.root
```

I see numerous warning messages like this (>1000 for 100 events):
```
Warning: Element Requested 64766 too large for Pt LUT (JET-ET) size = 2048
```

Either something in the L1 step should be fixed, or the L1 step should be removed from the premix stage 1 workflow.

attn: @mdhildreth 

\* e.g. `/store/relval/CMSSW_9_0_0_pre5/RelValMinBias_13/GEN-SIM/90X_upgrade2018_realistic_v12-v1/00000/047A850C-8400-E711-AB57-0CC47A7C3412.root`

**Comment by slava77** [comment]
assign l1

I noticed these as well. Can be pretty noisy at higher PU

@rekovic @mulhearn please clarify on ETA for resolution

**Comment by slava77** [comment]
minimally,
these couts in L1Trigger/L1TGlobal/src/GlobalScales.cc should be replaced with LogWarnings or LogInfos.

**Comment by kpedro88** [comment]
I no longer see these messages in IB tests (e.g. for workflow 250199.17), but for the future, the suggestion from @slava77 should probably still be followed.

**Comment by kpedro88** [comment]
closing this:
* couts were fixed in #19155
* L1 step no longer included in premix_stage1
